content/en/mobile_testing/mobile_app_tests/steps.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Mobile App Test Steps
+kind: documentation
+description: Learn how to automatically record and manually set steps in a mobile test recording.
+further_reading:
+- link: "/mobile_testing/mobile_app_tests/"
+  tag: "Documentation"
+  text: "Learn about Synthetic mobile tests"
+- link: "/mobile_testing/mobile_app_tests/advanced_options"
+  tag: "Documentation"
+  text: "Learn about advanced options in mobile tests"
+---
+
+## Overview
+
+Steps represent individually recorded interactions or assertions that you want to execute in your test. To define a step, click **Start Recording** and interact with the device as you would normally, or create a step manually by clicking **Assertions** or **Special Actions**. 
+
+## Launch a device
+
+To start recording and adding steps, select a device to launch a mobile app test from the dropdown menu and click **Launch Device**. 
+
+{{< img src="mobile_testing/launch_device.png" alt="Select a device to run a mobile test on" style="width:60%" >}}
+
+Select **Show only available devices. Available devices load faster** to see the most available devices for shorter wait times. Click the **Device Connection Notification** button to enable notifications for when your device is ready.
```

Comment:
"To show only devices that are currently available for testing select **Show only available devices**. Available devices load faster** to see the most available devices for shorter wait times." 

Can we create a new section for Notifications? Then we can specifically say what turning on notifications does i.e. it lets you know when your device is ready and when your device is going to timeout because of inactivity. 

We can include screenshots of what each of these notifications look like.

Reply:
```suggestion
Select **Show only available devices. Available devices load faster** to see the most available devices for shorter testing wait times. 

### Notifications

Click the **Device Connection Notification** button to enable notifications for when your device is ready and when your device is going to timeout because of inactivity.
```